ceno_emul/src/syscalls.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{RegIdx, Tracer, VMState, Word, WordAddr, WriteOp};
+use anyhow::Result;
+use itertools::chain;
+
+mod keccak_permute;
+
+pub const KECCAK_PERMUTE: u32 = 0x00_01_01_09;
```

Comment:
Please consider setting up a syscall numbers table in a common crate that both guest and host code can import.

(There's a few more such common data and structures that we might want to put in this simple crate.)

Reply:
This will evolve into something like this yes.